src/picongpu/include/plugins/adios/ADIOSWriter.hpp

```diff
@@ -421,10 +421,14 @@ class ADIOSWriter : public ILightweightPlugin
              (&mThreadParams.adiosAggregators)->default_value(0), "Number of aggregators [0 == number of MPI processes]")
             ("adios.ost", po::value<uint32_t > (&mThreadParams.adiosOST)->default_value(1),
              "Number of OST")
+            ("adios.disable-meta", po::bool_switch (&mThreadParams.adiosDisableMeta)->default_value(false),
```

Comment:
that's the case in *all* program options ;) someone must have started that madness ^^